components/eventing-controller/api/v1alpha1/subscription_conversion.go

```diff
@@ -1,19 +1,340 @@
 package v1alpha1
 
 import (
+	"fmt"
+	"strconv"
+	"strings"
+
 	"github.com/kyma-project/kyma/components/eventing-controller/api/v1alpha2"
+	"github.com/pkg/errors"
 	"sigs.k8s.io/controller-runtime/pkg/conversion"
 )
 
+const (
+	hubVersionErrMsg     = "hub version is not the expected v1alpha2 version"
+	multipleSourceErrMsg = "subscription contains more than 1 eventSource"
```

Comment:
Should we prefix this ? To make errors easily discoverable to the API users.

```suggestion
	errorHubVersionErrMsg     = "hub version is not the expected v1alpha2 version"
	errorMultipleSourceErrMsg = "subscription contains more than 1 eventSource"
```

Alternative would be to provide our own error package.